include/swift/SIL/SILInstruction.h

```diff
@@ -4729,50 +4667,6 @@ class ReleaseValueAddrInst
   }
 };
 
-/// Copies a loadable value in an unmanaged, unbalanced way. Only meant for use
-/// in ownership qualified SIL. Please do not use this EVER unless you are
-/// implementing a part of the stdlib called Unmanaged.
-class UnmanagedRetainValueInst
-    : public UnaryInstructionBase<SILInstructionKind::UnmanagedRetainValueInst,
-                                  RefCountingInst> {
-  friend SILBuilder;
-
-  UnmanagedRetainValueInst(SILDebugLocation DebugLoc, SILValue operand,
-                           Atomicity atomicity)
-      : UnaryInstructionBase(DebugLoc, operand) {
-    setAtomicity(atomicity);
-  }
-};
-
-/// Destroys a loadable value in an unmanaged, unbalanced way. Only meant for
-/// use in ownership qualified SIL. Please do not use this EVER unless you are
-/// implementing a part of the stdlib called Unmanaged.
-class UnmanagedReleaseValueInst
-    : public UnaryInstructionBase<SILInstructionKind::UnmanagedReleaseValueInst,
-                                  RefCountingInst> {
-  friend SILBuilder;
-
-  UnmanagedReleaseValueInst(SILDebugLocation DebugLoc, SILValue operand,
-                            Atomicity atomicity)
-      : UnaryInstructionBase(DebugLoc, operand) {
-    setAtomicity(atomicity);
-  }
-};
-
-/// Transfers ownership of a loadable value to the current autorelease
-/// pool. Unmanaged, so it is ignored from an ownership balancing perspective.
-class UnmanagedAutoreleaseValueInst
-    : public UnaryInstructionBase<SILInstructionKind::UnmanagedAutoreleaseValueInst,
-                                  RefCountingInst> {
-  friend SILBuilder;
-
-  UnmanagedAutoreleaseValueInst(SILDebugLocation DebugLoc, SILValue operand,
-                                Atomicity atomicity)
-      : UnaryInstructionBase(DebugLoc, operand) {
-    setAtomicity(atomicity);
-  }
-};
-
```

Comment:
Okay, these instructions are just confusingly named — or really, `UnmanagedStorageType` is.  At any rate, they have nothing to do with UnmanagedStorageType and should not be swept up in your changes.

Reply:
I've removed this in a forthcoming pull request update.